tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<ul><li>Write more descriptions about TensorList. In particular, look at the docstrings of <code>TensorList.h</code></li><li class="ql-indent-1">In particular, write about the motivation of the TensorList (e.g. store a list of 3d points, colors in a contiguous manner)</li><li>Move this part to the very end since it's it is about tensorlist not tensor.</li><li>The most important syntax of TensorList is to let user understand whether the underlying memory is copied or shared.</li><li class="ql-indent-1"><code>from_tensor</code>  is copied</li><li class="ql-indent-1"><code>from_tensors</code> there is a flag to control inplace or not</li><li class="ql-indent-2">When a TensorList is inplace, it can not be resized, extended, cleared, etc. See <code>TensorList.cpp</code>  to see what is the limitation of a inplace tensor.</li><li class="ql-indent-1">copy constructor always give "shallow" copy, give examples</li><li class="ql-indent-1">if an explicit copy is needed, show how the <code>Copy()</code>  function can be used.</li><li>Properties of Tensorlist: <a href="https://github.com/intel-isl/Open3D/blob/master/cpp/pybind/core/tensorlist.cpp#L97" target="_blank">https://github.com/intel-isl/Open3D/blob/master/cpp/pybind/core/tensorlist.cpp#L97</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>